src/node.cc

```diff
@@ -2935,6 +2942,12 @@ static void PrintHelp() {
          "  -p, --print          evaluate script and print result\n"
          "  -i, --interactive    always enter the REPL even if stdin\n"
          "                       does not appear to be a terminal\n"
+#ifdef _WIN32
+         "  -r, --require        ';'-separated list of modules to"
```

Comment:
I don't think we need this... you can use `:` in unix or win32 package names (unadvised, though, and if it fails to work your bug reports will be laughed at). Stick with `:`. Thoughts, @piscisaureus, or @orangemocha, am I too unix-centric? :-)